modules/cnv-uploading-local-disk-image-pvc.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-uploading-local-disk-images-virtctl.adoc
+
+[id="cnv-uploading-local-disk-image-pvc_{context}"]
+= Uploading a local disk image to a new PersistentVolumeClaim
+
+You can use the `virtctl` CLI utility to upload a virtual machine disk image from
+a client machine to your cluster. This creates a PersistentVolumeClaim (PVC)
```

Comment:
~This creates a PersistentVolumeClaim (PVC)
that can be associated with a virtual machine after the upload has completed.~
Uploading the disk image creates a PersistentVolumeClaim (PVC) that you can
associate with a virtual machine.